src/main/java/blackjack/domain/participant/Players.java

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain.participant;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Players {
+
+    private static final int MIN_PLAYER_SIZE = 1;
+    private static final int MAX_PLAYER_SIZE = 4;
+
+    private final List<Player> players;
+
+    Players(List<Player> players) {
+        this.players = players;
+    }
+
+    static public Players from(List<String> playerNames, List<Integer> bettings) {
```

Comment:
```suggestion
    public static Players from(List<String> playerNames, List<Integer> bettings) {
```
public 을 static 보다 먼저 붙이는 습관을 들여보죠!

Reply:
저게 컴파일 되네요... 잘 확인해보겠습니다.